ChustaSoft.GamersPlatformUtils.Domain.Origin/Implementations/OriginBusiness.cs

```diff
@@ -1,39 +1,29 @@
 ﻿using ChustaSoft.GamersPlatformUtils.Abstractions;
-using ChustaSoft.GamersPlatformUtils.Domain.Constants;
 using Microsoft.VisualBasic.FileIO;
 using Microsoft.Win32;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Runtime.InteropServices;
-using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
-namespace ChustaSoft.GamersPlatformUtils.Domain.Implementations
+namespace ChustaSoft.GamersPlatformUtils.Domain
 {
-    public class OriginBusiness : PlatformBase, ILinkFinder, IAnalyzer
+    public class OriginBusiness : PlatformBase, IOriginBusiness, ILinkFinder, IAnalyzer
```

Comment:
As IOriginBusiness is now inheriting from ILinkFinder and IAnalizer, should we remove ILinkFinder and IAnalizer from here?

Reply:
Same above...